ztp/gitops-subscriptions/argocd/Upgrade.md

```diff
@@ -1,43 +1,127 @@
-## Steps to Upgrade from v4.9 [argocd/example](https://github.com/openshift-kni/cnf-features-deploy/tree/release-4.9/ztp/gitops-subscriptions/argocd/resource-hook-example) to v4.10 [argocd/example](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/gitops-subscriptions/argocd/example)
+## Upgrading GitOps ZTP
```

Comment:
I believe its better to keep the upgrade version v4.9 to v4.10 , we may have another section in Upgrade.md from 4.10 to 4.11